echo.js

```diff
@@ -1,7 +1,25 @@
-function echo(){
-  // ...
-}
+onload = function() {
+  // echo function is to print out each argument on a new line
+  function echo() {
 
-echo();
-echo('bla');
-echo('foo', 'bar', 'baz');
+    var createElement = document.createElement('p');
+    var innerContent = [];
+    var count = 0;
+    //if arguments is not zero
+    if (arguments.length !== 0) {
+      for (var i in arguments) {
```

Comment:
Careful with using `for...in` with arrays (or `arguments`).

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in#Array_iteration_and_for...in